modules/aws/eks/iam.tf

```diff
@@ -257,3 +257,19 @@ module "trivy_ecr" {
   kubernetes_service_account = "trivy"
   policy_json                = data.aws_iam_policy_document.starboard_ecr_read_only.json
 }
+
+module "eks_ebs_csi_driver" {
+  source = "../irsa"
+
+  name = "${var.name_prefix}-${data.aws_region.current.name}-${var.environment}-${var.name}${var.eks_name_suffix}-eks_ebs_csi"
+  oidc_providers = [
+    {
+      url = aws_iam_openid_connect_provider.this.url
+      arn = aws_iam_openid_connect_provider.this.arn
+    }
+  ]
+  kubernetes_namespace         = "kube-system"
+  kubernetes_service_account   = "ebs-csi-controller-sa"
+  policy_json                  = data.aws_iam_policy_document.cluster_autoscaler.json
```

Comment:
Is this on purpose of a mistake?

Reply:
Hmm I screwed up some rebase or something, I thought I removed it after I did somet testing. 
I will update it.

Reply:
I suppose `policy_json` should be empty considering you made it optional?

Reply:
I had fixed it, but I hadn't pushed it ^^